.codeclimate.yml

```diff
@@ -1,8 +1,18 @@
-version: "2"         # required to adjust maintainability checks
+version: "2" # required to adjust maintainability checks
 checks:
   method-lines:
     config:
       threshold: 80
+plugins:
```

Comment:
Added ESLint to code climate, now the eslint errors where only shown in the github actions summary, which were not really visible. Now it is part of a 'code climate review'